### ISSUE TITLE
starting offset

### ISSUE THREAD

**Issue by fullermetric** [simonsmith/jquery.stickyTableHeader#6]
Nice plugin, but there doesn't appear to be the option to give a starting offset for the table.  I use another plugin for sticky headers which provides this option, but it breaks the table formatting if I use their plugin over yours.  If I can figure out how to modify your code and get it to work I will add it here.

**Comment by simonsmith** [comment]
Do you mean you would want the `top` on the `position: fixed` to be higher than `0`?

**Comment by fullermetric** [comment]
Hi Simon. Yes we want the top position to be a flexible offset.  We have a wordpress site that already uses a sticky header plugin freezing the top 160px for navbar reasons, search window, etc. 

If your plugin offered the ability to take in an offset it would help us out.

I would have made the changes myself, but I'm a neophyte when it comes to building packages using npm.  I tried following your instructions.  I installed npm, downloaded and extracted zip file from your Github repo.  I copied this folder to my node_modules folder. ie.
**C:\Users\<computername>\node_modules\jquery.stickyTableHeader-master** 

Dropped into cmd and changed directory to that folder.
Typed in npm run build, but got a bunch of error message that I don't need to throw your way.

I'm hoping you can make the change and offer us a stickyTableHeader.min.js on your github account. 
That is something I think would help out those of us lacking npm building knowledge and/or ES6 knowledge.

Thanks for asking.

**Comment by simonsmith** [comment]
@fullermetric No problem. I'm happy to make the change, it's a reasonable request. Will take a look when I can

**Comment by simonsmith** [comment]
Try `1.1.0` and let me know

**Comment by fullermetric** [comment]
Hi Simon,

Unfortunately, I went with another Sticky Header plugin for jQuery.  It wasn’t that I couldn’t wait for the enhancement
but had to do with the “sticky” table header worked intermittently.  It was fine for ½ a day when I was testing it.  When
I came back from lunch it wasn’t working.  Not sure if it was because I was testing this is a busy network environment?
II tried playing with the throttling set it as  low as 10ms, but to no avail.  Funny thing, I tried set breakpoints in the plugin code
using  chrome dev tools and it started working again until I removed the breakpoints and it stopped working.

I will keep your plugin in mind and try it again with a different project.

K.
